vuu-ui/packages/vuu-ui-controls/src/combo-box/ComboBox.tsx

```diff
@@ -207,7 +213,7 @@ export const ComboBox = forwardRef(function Combobox<
           highlightedIndex={highlightedIndex}
           itemTextHighlightPattern={String(inputProps.value) || undefined}
           listHandlers={listHandlers}
-          onSelectionChange={onSelectionChange}
+          onSelectionChange={onSelectionChange} // not really needed, since onClick in listHandlers will be used instead.
```

Comment:
@heswell let me know if I should get rid of this, I've checked it's not used if `listHandlers` is not undefined. And it is not undefined here.

Reply:
yes, lets get rid of it